resources/eng.ini

```diff
@@ -195,7 +194,7 @@ warp.success=You are teleported to plot {%0}
 
 ; The /p middle Subcommand
 middle.name=middle
-middle.alias=mid
+middle.alias=middle
```

Comment:
This needs reverted. mid is the alias.

Reply:
Oh lol, I thougt I edit something else 🤦‍♂